certified-connectors/Asite/apiProperties.json

```diff
@@ -0,0 +1,55 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Microsoft Power Automate",
+          "scopes": [
+            "DOCUMENT_READ DOCUMENT_WRITE"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
```

Comment:
@asitedeveloper The `reidrectURL` property can be removed if your have already mentioned `"redirectMode": "Global"`

Reply:
@sriyen-msft We are removed redirectUrl property in apiProperties.json file.